example_dags/example_google_bigquery_gcs_load_and_save.py

```diff
@@ -0,0 +1,53 @@
+"""
+This Example DAG:
+ - Pulls a CSV file from Github and loads it into BigQuery.
+ - Extracts the data from BigQuery and load into in-memory Pandas Dataframe
+ - Finds the Top 5 movies based on Rating using pandas dataframe
+ - And loads it into a Google Cloud Storage bucket in a CSV file
+
+Pre-requisites:
```

Comment:
It may be worth to add to the pre-requisites the fact the user needs to install astro either using `astro[all]` or `astro[google]`.

Reply:
Done on L9: I didn't add the library name purposely, so we don't have to rename it when we change the package name

Reply:
https://github.com/astro-projects/astro/blob/caa57e17db01b402ce6ca33ff5e655590b684650/example_dags/example_google_bigquery_gcs_load_and_save.py#L9